src/services/devHandler/triggerTopicAutoMessage.js

```diff
@@ -0,0 +1,16 @@
+const {successResponse, errorResponse} = require('../../utils');
+const {topicAutoMessageProcess} = require('../../processes/topic-auto-msg-process');
+
+const triggerTopicAutoMessage = async (req, res) => {
+  try {
+    const result = await topicAutoMessageProcess();
+    return successResponse(res, 'triggerTopicAutoMessage succesfully', result);
+  } catch (error) {
+    console.log(error);
+    return errorResponse(res, error.toString(), 500);
+  }
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The error handling in this function could be improved. It's generally not a good practice to log the error directly to the console because it can expose sensitive information. Instead, consider using a logging library that can mask sensitive data and provide more context about the error. Also, you should not convert the error to string when sending it in the response. This could potentially leak sensitive information to the client. Instead, send a generic error message to the client and log the actual error on the server side for debugging.

```diff
const triggerTopicAutoMessage = async (req, res) => {
  try {
    const result = await topicAutoMessageProcess();
    return successResponse(res, 'triggerTopicAutoMessage succesfully', result);
  } catch (error) {
-    console.log(error);
-    return errorResponse(res, error.toString(), 500);
+    logger.error(error); // Use a logger like winston or bunyan
+    return errorResponse(res, 'An error occurred while triggering the auto message', 500);
  }
};
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->